common/nodejs-utils/src/package_json.rs

```diff
@@ -25,6 +25,18 @@ pub struct PackageJson {
     pub package_manager: Option<PackageManager>,
 }
 
+impl PackageJson {
+    #[must_use]
+    pub fn has_dependencies(&self) -> bool {
+        for dep_group in [self.dependencies.as_ref(), self.dev_dependencies.as_ref()] {
+            if dep_group.map_or_else(|| false, |deps| !deps.is_empty()) {
+                return true;
+            }
+        }
+        false
```

Comment:
I had to stop to unpack the `map_or_else` with the return body logic.  May I suggest the following refactor instead?

```suggestion
        let no_deps = !self.dependencies.as_ref().is_some_and(HashMap::is_empty);
        let no_dev_deps = !self
            .dev_dependencies
            .as_ref()
            .is_some_and(HashMap::is_empty);
        !no_deps || !no_dev_deps
```

Reply:
Thanks - I ended up deleting this function in the latest commit anyways.

Reply:
Whoops, I didn't delete this one. But I did like your `.any` suggestion, which I applied in 44e0adf.